static/.htaccess

```diff
@@ -1,16 +0,0 @@
-# Set far-future Expires headers for static media
-ExpiresActive on
-
-ExpiresDefault "access plus 1 week"
-ExpiresByType text/css "access plus 1 year"
-ExpiresByType text/javascript "access plus 1 year"
-ExpiresByType image/png "access plus 1 month"
-ExpiresByType image/jpeg "access plus 1 month"
-ExpiresByType image/gif "access plus 1 month"
-ExpiresByType image/vnd.microsoft.icon "access plus 1 month"
-ExpiresByType video/webm "access plus 1 week"
-ExpiresByType video/ogg "access plus 1 week"
-ExpiresByType video/x-flv "access plus 1 week"
-ExpiresByType application/x-shockwave-flash "access plus 1 week"
```

Comment:
Glad you brought this up. I wasn't sure about this. The WhiteNoise docs say the following about expiration values:

> The default is chosen to be short enough not to cause problems with stale versions but long enough that, if you’re running WhiteNoise behind a CDN, the CDN will still take the majority of the strain during times of heavy load.

Does that mean WhiteNoise will choose some reasonable defaults for us?